src/device/mod.rs

```diff
@@ -8,16 +8,16 @@ pub mod drop_privileges;
 mod integration_tests;
 pub mod peer;
 
-#[cfg(any(target_os = "macos", target_os = "ios"))]
+#[cfg(any(target_os = "macos", target_os = "freebsd"))]
```

Comment:
was that `target_os = "ios"` not load bearing?

Reply:
No, I'm afraid ios is never gonna support `utun` :(